server/server.js

```diff
@@ -1,11 +1,43 @@
 const express = require('express');
 const path = require('path');
+const bodyParser = require('body-parser');
```

Comment:
It doesn't look like you are actually using the 'body-parser' module, were you planning to use it later on?

Reply:
You're right!  I would need it for a POST request, but I haven't implemented that yet, so it shouldn't be there.  I'll take out first thing as I continue.